doc/devdocs/modules/powerpreview/monaco/readme.md

```diff
@@ -0,0 +1,15 @@
+# Developer Preview (Monaco)
+## Update monaco editor
+
+1. Download Monaco editor with npm: `npm i monaco-editor`.
+2. Delete everything except the `min` folder (the minimised code).
+3. Copy the `min` folder inside the [`monacoSRC`](/src/modules/previewpane/MonacoPreviewHandler/monacoSRC) folder.
```

Comment:
Why do we only copy min folder?!

Reply:
Because we only need the files in this folder.